docs/reference/motlp.md

```diff
@@ -173,11 +175,34 @@ When creating a Kubernetes secret, always encode the full string in Base64, incl
 
 ::::::
 
+% ## Rate limiting and throttling
+% 
+% The following rate limits apply to the {{motlp}}:
+% 
+% * n requests per second per API key
+% * m requests per minute per API key
+% 
+% If you exceed these limits, you will receive an HTTP 429 status code.
```

Comment:
We should have a section about, Please reach out to support and then Engineering can look in to Increasing it based on the License tier or experimentation purposes.